### ISSUE TITLE
docker dosn't start

### ISSUE THREAD

**Issue by PROsalamoz** [docker/for-mac#5770]
- [x] I have tried with the latest version of Docker Desktop
  - [ ] I have tried disabling enabled experimental features
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: 228499B9-F9F1-443B-8355-56D8452E3AEA/20210620214732

### Expected behavior
- start
### Actual behavior
- crash Fatal Error
### Information

  - macOS Version: 11.4
  - Intel chip or Apple chip: intel chip
  - Docker Desktop Version: 3.4.0

### Steps to reproduce the behavior
just click on the 
<img width="666" alt="‏لقطة الشاشة ٢٠٢١-٠٦-٢٠ في ١١ ٥٥ ٥٩ م" src="https://user-images.githubusercontent.com/42225950/122689596-210ba900-d224-11eb-93b3-6ec9a21a9691.png">
<img width="938" alt="‏لقطة الشاشة ٢٠٢١-٠٦-٢٠ في ١١ ٤٨ ٤٤ م" src="https://user-images.githubusercontent.com/42225950/122689601-2537c680-d224-11eb-8d51-2d467e8df0f7.png">
docker icon

  1. ...
  2. ...

**Comment by mitryushina** [comment]
I have the same error while starting:
![image](https://user-images.githubusercontent.com/83217573/122732489-01589d00-d285-11eb-9125-cecde22fac89.png)

macOS Version: 11.4
Intel chip or Apple chip: intel chip

Diagnostics ID: 9F1BB79B-6B88-45C6-872F-D2F356C5CC5A/20210621082654

**Comment by djs55** [comment]
Thanks for the report. It might be a $PATH issue. Could you try the following command from a terminal:
```
 open --env PATH=/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin /Applications/Docker.app
```

**Comment by mitryushina** [comment]
The command  `open --env PATH=/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin /Applications/Docker.app` helped, docker started. Thanks!

The commands' results:

**uname -a**
```
Darwin nmitryushina 20.5.0 Darwin Kernel Version 20.5.0: Sat May  8 05:10:33 PDT 2021; root:xnu-7195.121.3~9/RELEASE_X86_64 x86_64

```
**which uname**
```
/usr/bin/uname
```

**Comment by PROsalamoz** [comment]
@djs55 

thanks a lot,
The first command help to start docker

<img width="857" alt="commands" src="https://user-images.githubusercontent.com/42225950/122794616-bce8f380-d2bc-11eb-8186-9e3fd261f7e9.png">

**Comment by djs55** [comment]
@mitryushina @PROsalamoz I'm glad to hear that Docker now starts! Clearly there is a $PATH problem, let me see if I can fix it in Docker itself.

**Comment by djs55** [comment]
@mitryushina @PROsalamoz If you would like to try it, I have a developer build with a candidate fix for this: https://desktop-stage.docker.com/mac/stable/amd64/65968/Docker.dmg

**Comment by StefanScherer** [comment]
We have just released Docker Desktop 3.5.0 which contains a fix for this issue. I'll close this ticket now, but feel free to comment and give us feedback. If there is still a problem, please open a new ticket.
